spec/glueby/internal/wallet_spec.rb

```diff
@@ -73,6 +73,33 @@ def load_wallet(wallet_id); end
 
   end
 
+  describe 'broadcast' do
+    let(:tx) { Tapyrus::Tx.new }
+    let(:wallet) { Glueby::Internal::Wallet.create }
+
+    context 'A block argument is given' do
+      let(:block) { Proc.new {} }
+
+      it 'pass the block to a wallet adapter' do
+        expect(Glueby::Internal::Wallet.wallet_adapter)
+          .to receive(:broadcast).with('created_wallet_id', tx) do |*args, &proc|
+          expect(proc).to eq(block)
+        end
```

Comment:
You should also verify that tx is an argument to block.

Reply:
I added unit tests at ad09d9021b0241dad85ea25461639c92fa3acd65